cmd/util/exitcodes.go

```diff
@@ -0,0 +1,21 @@
+package util
+
+const (
+	// ExitSuccess Successful termination
+	ExitSuccess = 0
+
+	// ExitError Catchall for general errors
+	ExitError = 1
+
+	// ExitBuiltinMisuse Misuse of shell builtins (according to Bash documentation)
```

Comment:
This is a bash-ism, there are some POSIX defined error codes I believe but it is otherwise surprisingly non-standard.  The best we can do for now is probably using the Linux error codes that should be listed in the man page for errno `man errno` (they are in osx)

Reply:
These are no longer used, so will just remove the file.